### PR TITLE
2265 projection tools

### PR DESCRIPTION
**Pull request by rppawlo**
Projection tools

@trilinos/panzer

### CODE REVIEW

**Comment by rppawlo** [comment]
verified this is passing on cuda as well

**Comment by jmgate** [comment]
Is there anything particular you wanted me to focus on here?  I'm afraid my brain is fried from a week of a conference + cold, so all I'm seeing is green and red lines of code blurring together.

**Comment by rppawlo** [comment]
@jmgate - nothing in particular. Just looking for anyone to approve so I can push. Tests passed the PR testing and I also ran tests on cuda.

File: packages/panzer/disc-fe/src/Panzer_L2Projection.cpp [hunk 1]
```diff
@@ -0,0 +1,8 @@
+// @HEADER
+// @HEADER
+
+#include "PanzerDiscFE_config.hpp"
+#include "Panzer_L2Projection.hpp"
+#include "Panzer_L2Projection_impl.hpp"
+
+template class panzer::L2Projection<int,panzer::Ordinal64>;
```
Comment (sconde) [inline packages/panzer/disc-fe/src/Panzer_L2Projection.cpp]: Can you build this for `<int,int>` too?

Comment (rppawlo) [reply packages/panzer/disc-fe/src/Panzer_L2Projection.cpp]: yes - panzer::Ordinal64 is a configure time chosen integer type that is set based on what Tpetra supports (if being built with ETI).

**Comment by rppawlo** [comment]
Thanks @jmgate !

**Pull request merged**